Assets/Scripts/Game/UserInterface/UserInterfaceWindow.cs

```diff
@@ -128,7 +128,7 @@ public void CloseWindow()
         {
             uiManager.PopWindow();
             RaiseOnCloseHandler();
-            DaggerfallGC.ThrottledUnloadUnusedAssets();
+            //DaggerfallGC.ThrottledUnloadUnusedAssets();
```

Comment:
Did you mean to keep this comment? Should we remove the line instead?

Reply:
Line is now removed.